tools/rna_tools/mlocarna/mlocarna.xml

```diff
@@ -0,0 +1,167 @@
+            </param>
+            <when value="global_locarna">
+                <param name="free_endgaps" type="select" label="Free endgaps">
+                    <option value="">No free endgaps</option>
+                    <option value="--free-endgaps">Free endgaps</option>
+                    <option value="--free-endgaps-5">Free endgaps, only 5'</option>
+                    <option value="--free-endgaps-3">Free endgaps, only 3'</option>
+                </param>
+            </when>
+            <when value="local_locarna" />
+            <when value="probabilistic">
+                <param name="consistency_transformation" type="boolean" 
+                       truevalue="--consistency-transformation" falsevalue="" 
+                       checked="true" label="Consistency transformation" 
+                       help="--consistency-transformation" />
+                <param name="iterate" type="boolean"
+                       truevalue="true" falsevalue="false"
+                       checked="false" label="Iterative refinement"
+                       help="--iterate" />
+                <param name="iterations" type="integer"
+                       value="1" label="Number of refinement iterations"
+                       help="--iterations num" />
+            </when>
+            <when value="sparse" />
+        </conditional>
+        
+        <section name="scoring" title="Scoring parameters">
+            <expand macro="common_scoring_parameters" />
+        </section>
+        
+        <section name="folding" title="RNA folding parameters">
+            <expand macro="common_folding_parameters" />
+        </section>
+
+        <section name="heuristics" title="Heuristic parameters">
+            <expand macro="common_heuristic_parameters" />
+        </section>
+
+        <section name="other" title="Other parameters">
+            <expand macro="common_other_parameters" />
+            <param name="lonely_pairs" type="boolean" truevalue="--LP" falsevalue="--noLP" 
+                   checked="false" label="Allow lonely base-pairs" help="(--LP/--noLP)" />
+        </section>
+    </inputs>
+    
+    <outputs>
+        <expand macro="common_outputs" />
+    </outputs>
+    
+    <tests>
+        <test>
+            <param name="input_data" value="archaea.fa" />
+            <output name="stdout" file="archaea-default.stdout" />
+        </test>
+        <test>
+            <param name="input_data" value="haca.snoRNA.fa" />
+            <output name="stdout" file="haca.snoRNA-default.stdout" />
+        </test>
+        <test>
+            <param name="input_data" value="archaea.fa" />
+            <param name="alignment_mode_selector" value="probabilistic" />
+            <output name="stdout" file="archaea-probabilistic.stdout" />
+        </test>
+    </tests>
+
+    <help><![CDATA[ **Multiple alignment of RNAs**
+    
+For more information, see     
```

Comment:
I would prefer some more help directly in the tool, for people without internet access.
But we can do this later.